sqlglot/dialects/dialect.py

```diff
@@ -0,0 +1,252 @@
+import importlib
+
+from sqlglot import exp
+from sqlglot.generator import Generator
+from sqlglot.helper import csv, list_get
+from sqlglot.parser import Parser
+from sqlglot.time import format_time
+from sqlglot.tokens import Tokenizer
+from sqlglot.trie import new_trie
+
+
+
+class _Dialect(type):
+    classes = {}
+
+    @classmethod
+    def import_dialect(cls, key):
+        if key not in cls.classes:
+            importlib.import_module(f"sqlglot.dialects.{key}")
```

Comment:
I don't like the idea of hardcoding the sqlglot.dialects. here? Can't we just use the relative path instead?
```
importlib.import_module(f".{key}")

Reply:
no, you can't do that, i'd have to include the package

Reply:
Can we use `sqllib.dialect.__name__` then?

Reply:
my suggestion was to always import at init and drop the lazy load